### ISSUE TITLE
Accessibility fixes for docs site: Add skip to main content link

### ISSUE THREAD

**Issue by GeriReid** [newscorp-ghfb/NewsKit#737]
# Description
Including a skip link gives users the option to bypass the top-level navigation links and jump to the main content on a page. This helps users who navigate using the keyboard get to the content quickly.

https://webaim.org/techniques/skipnav/

# Tasks/Steps
* Create a “Skip to content” button component in site/. From HTML point of view should be an <a> tag, pointing to the main content of the page through a #link.For TADo we want the button to appear in every page? if yes how do we know where to point take the use every time? do we always have a main element? or something to take the user to? or it needs to change per page, using a config file with info?

* Should be the first focusable element on the page (or after the logo) Designers to provide a design on where this button should appear on the page and content)

* Visually hidden until receives focus from tabbing

* Affects all pages

# Acceptance Criteria
* Using only the keyboard, use the link to move focus to the main content of the page

* After moving focus to the main content, the next tab should move you to the first interactive element on the page

* Review with Design
